src/Game.js

```diff
@@ -7,17 +7,17 @@ class Game {
     this.player1 = new Player(allNames[0], 1);
     this.player2 = new Player(allNames[1], 2);
     this.player3 = new Player(allNames[2], 3);
-    this.round = new Round(this.randomizeCategories());
+    this.round = new Round(this.returnRandomizedCategories());
   }
 
-  randomizeCategories() {
+  returnRandomizedCategories() {
     for (var i = this.categories.length - 1; i > 0; i--) {
       var j = Math.floor(Math.random() * (i + 1));
       var temp = this.categories[i];
       this.categories[i] = this.categories[j];
```

Comment:
We could make use of array destructuring to refactor the portion where variables are being swapped.